kie-remote/kie-remote-services/src/main/java/org/kie/remote/services/jaxb/JaxbCommandsResponse.java

```diff
@@ -212,8 +218,10 @@ public void addResult(Object result, int i, Command<?> cmd) {
             this.responses.add(new JaxbVariableInstanceLog((VariableInstanceLog) result));
         } else if( result instanceof DefaultFactHandle ) { 
            this.responses.add(new JaxbOtherResponse(result, i, cmd));
-        } else if( cmd instanceof GetTaskContentCommand || cmd instanceof GetContentMapForUserCommand ) { 
-           this.responses.add(new JaxbTaskContentResponse((Map<String, Object>) result, i, cmd));
+        } else if( cmd instanceof GetTaskContentCommand || cmd instanceof GetContentMapForUserCommand ) {
+            this.responses.add(new JaxbTaskContentResponse((Map<String, Object>) result, i, cmd));
+        } else if ( cmd instanceof GetTaskByWorkItemIdCommand ) {
+            this.responses.add(new JaxbTaskResponse((Task)result, i,  cmd));
```

Comment:
Thanks, I fixed both. Please check the result.

I had to change `this.responses.add(new JaxbTaskResponse((JaxbTask)result, i, cmd));` to `this.responses.add(new JaxbTaskResponse((Task)result, i, cmd));` because the result is TaskImpl in case of the `GetTaskByWorkItemCommand`